index.js

```diff
@@ -65,39 +65,67 @@ var findFixmes = function(file){
   })
 }
 
-var eligibleFile = function(fp, excludePaths){
-  return (excludePaths.indexOf(fp.split("/code/")[1]) < 0) &&
-  !fs.lstatSync(fp).isDirectory() &&
-  (excludeExtensions.indexOf(path.extname(fp)) < 0)
+// Returns an array of unique array values not included in the other provided array
+var diff = function(a1, a2) {
```

Comment:
What do you think of putting this in its own module and requiring it at the top of the file?